pacta/README.md

```diff
@@ -44,9 +44,28 @@ If the build is successful, load the image interactively and push the tags
 created for each of the PACTA_analysis and friends repos inside of the
 docker container.
 
+# Releasing
 
-# For the web
+To release a new version of the software, push all tags to GitHub. The tag you provide will affect the image, the repository containing the dockerfile that builds the image, and the pacta siblings. That consistent tag make the process reproducible.
+
+To push the tags, start the container with something like
+
+```bash
+cd ~
+docker run --rm -ti -v "$(pwd)/.ssh":/root/.ssh 2dii_pacta:latest
```

Comment:
at least on unix-likes, probably `"~/.ssh"` is safer?

Reply:
Yeah, but volumes like absolute paths -- I think

Reply:
ah, yes... haha I just ran into that... nvmd